modules/rostests/rosautotest/CJournaledTestList.cpp

```diff
@@ -154,7 +154,7 @@ CJournaledTestList::WriteInitialJournalFile()
     CTestInfo* TestInfo;
     DWORD BytesWritten;
 
-    StringOut("Writing initial journal file...\n\n");
+    StringOut("[ROSAUTOTEST] Writing initial journal file...\n\n");
```

Comment:
Can't we just have this done in StringOut?

Reply:
Not as is, see comments on #1531: that would probably need to create a `StringOutRAT()` or `StringOutTest()`...